contracts/imports/helper.fc

```diff
@@ -2,9 +2,10 @@
 
 cell fill_zeros(cell jetton_masters) {
     cell assets = new_dict();
+    slice zero_cs = begin_cell().store_int(0,257).end_cell().begin_parse(); 
```

Comment:
This can be implemented with function `slice zero_cs() asm "... PUSHSLICE";` to avoid even one `.end_cell()`.